jbpm-installer/src/test/resources/ddl60/mysql5/mysql5-jbpm-drop-schema.sql

```diff
@@ -0,0 +1,44 @@
+alter table Attachment drop foreign key FK1C93543D937BFB5;
+alter table Attachment drop foreign key FK1C9354333CA892A;
+alter table BooleanExpression drop foreign key FKE3D208C06C97C90E;
+alter table CorrelationPropertyInfo drop foreign key FK761452A5D87156ED;
+alter table Deadline drop foreign key FK21DF3E78A9FE0EF4;
+alter table Deadline drop foreign key FK21DF3E78695E4DDB;
+alter table Delegation_delegates drop foreign key FK47485D5772B3A123;
+alter table Delegation_delegates drop foreign key FK47485D57786553A5;
+alter table ErrorInfo drop foreign key FK8B1186B6724A467;
+alter table Escalation drop foreign key FK67B2C6B5D1E5CC1;
+alter table I18NText drop foreign key FK2349686BF4ACCD69;
+alter table I18NText drop foreign key FK2349686B424B187C;
+alter table I18NText drop foreign key FK2349686BAB648139;
+alter table I18NText drop foreign key FK2349686BB340A2AA;
+alter table I18NText drop foreign key FK2349686BF0CDED35;
+alter table I18NText drop foreign key FK2349686BCC03ED3C;
+alter table I18NText drop foreign key FK2349686B77C1C08A;
+alter table I18NText drop foreign key FK2349686B18DDFE05;
+alter table I18NText drop foreign key FK2349686B78AF072A;
+alter table Notification drop foreign key FK2D45DD0BC0C0F29C;
+alter table Notification_BAs drop foreign key FK2DD68EE072B3A123;
+alter table Notification_BAs drop foreign key FK2DD68EE093F2090B;
+alter table Notification_email_header drop foreign key FKF30FE3448BED1339;
+alter table Notification_email_header drop foreign key FKF30FE3443E3E97EB;
+alter table Notification_Recipients drop foreign key FK98FD214E72B3A123;
+alter table Notification_Recipients drop foreign key FK98FD214E93F2090B;
+alter table PeopleAssignments_BAs drop foreign key FK9D8CF4EC72B3A123;
+alter table PeopleAssignments_BAs drop foreign key FK9D8CF4EC786553A5;
+alter table PeopleAssignments_ExclOwners drop foreign key FKC77B97E472B3A123;
+alter table PeopleAssignments_ExclOwners drop foreign key FKC77B97E4786553A5;
+alter table PeopleAssignments_PotOwners drop foreign key FK1EE418D72B3A123;
+alter table PeopleAssignments_PotOwners drop foreign key FK1EE418D786553A5;
+alter table PeopleAssignments_Recipients drop foreign key FKC6F615C272B3A123;
+alter table PeopleAssignments_Recipients drop foreign key FKC6F615C2786553A5;
+alter table PeopleAssignments_Stakeholders drop foreign key FK482F79D572B3A123;
+alter table PeopleAssignments_Stakeholders drop foreign key FK482F79D5786553A5;
+alter table Reassignment drop foreign key FK724D056062A1E871;
+alter table Reassignment_potentialOwners drop foreign key FK90B59CFF72B3A123;
+alter table Reassignment_potentialOwners drop foreign key FK90B59CFF35D2FEE0;
+alter table Task drop foreign key FK27A9A53C55C806;
+alter table Task drop foreign key FK27A9A5B723BE8B;
+alter table Task drop foreign key FK27A9A55427E8F1;
+alter table task_comment drop foreign key FK61F475A57A3215D9;
+alter table task_comment drop foreign key FK61F475A5F510CB46;
```

Comment:
Here and in the `mysql-innodb-jbpm-drop-schema.sql` you removed the `drop table if exists` statements. Why is that? Also I noticed (purely by chance) that these 2 drop scripts are missing dropping of foreign key in `EventTypes` table. Although they are not needed because the referenced table is deleted after that, it would be good to include it.

Reply:
Restored lines, thanks!